src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -2536,7 +2536,7 @@ protected override void ProcessRecord()
                     try
                     {
                         resolvedPSPaths = SessionState.Path.GetResolvedPSPathFromPSPath(path, currentContext);
-                        if (true == SuppressWildcardExpansion && 0 == resolvedPSPaths.Count)
+                        if (SuppressWildcardExpansion == true && resolvedPSPaths.Count == 0)
```

Comment:
I wonder it seems we already fixed the bad patterns like `SuppressWildcardExpansion == true`

Reply:
We will fix in later PR #13454